plugins/elantp/fu-elantp-i2c-device.c

```diff
@@ -269,7 +357,19 @@ fu_elantp_i2c_device_setup(FuDevice *device, GError **error)
 	/* define the extra instance IDs (ic_type + module_id) */
 	fu_device_add_instance_u8(device, "ICTYPE", ic_type);
 	fu_device_build_instance_id(device, NULL, "ELANTP", "ICTYPE", NULL);
-	fu_device_build_instance_id(device, NULL, "ELANTP", "ICTYPE", "MOD", NULL);
+	if (g_strcmp0(fu_device_get_custom_flags(device), "elantp-absolute") == 0) {
```

Comment:
I'd much rather you use fu_device_register_private_flag() and fu_device_has_private_flag()

Reply:
OK, I will try to modify it.

Reply:
Many thanks, updated as suggested.